### PR TITLE
feat: update and add hooks

### PR DESCRIPTION
**Pull request by mr-tz**
also brings in some ideas from #307

### CODE REVIEW

File: scripts/render-ida-import-script.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def AppendComment(ea, string, repeatable=False):
     else:
         if string in current_string:  # ignore duplicates
             return
-        cmt = string + "\\n" + string
+        cmt = current_string + "\\n" + string
```
Comment (mr-tz) [inline scripts/render-ida-import-script.py:75]: unrelated fix, sorry

File: floss/main.py [hunk 2]
```diff
@@ -130,6 +130,7 @@ def make_parser(argv):
         "-n",
         "--minimum-length",
         dest="min_length",
+        type=int,
```
Comment (mr-tz) [inline floss/main.py:133]: unrelated fix, sorry

**Review by williballenthin — APPROVED** [review]
